--- textfunc.c	2006-11-06 09:59:12.000000000 +0100
+++ textfunc.c.oden	2009-10-05 13:53:54.000000000 +0200
@@ -227,7 +227,7 @@ void format_output (char *format_string,
 
 	while((percent=strchr(format,'%'))) {
 		*percent=0;
-		printf(format);
+		printf("%s", format);
 		*percent='%';
 		code=percent+1;
 		while(*code && (code[0] != '%' && !isalpha(*code))) code++;
@@ -354,7 +354,7 @@ void format_output (char *format_string,
 		}
 		
 	}
-	printf(format);
+	printf("%s", format);
 }
 
 
